.changeset/four-dolphins-report.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-permission-node': patch
+---
+
+Use new `PermissionEvaluator#query` method in `ServerPermissionClient` and test suites.
```

Comment:
We should call out the updated signature for `PermissionEvaluator#authorize` as a breaking change